### PR TITLE
SONAR-8339 drop access_log-like log debug line in Web server

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/platform/web/RootFilter.java [hunk 1]
```diff
@@ -50,26 +46,10 @@
...
-
-  private static final String MESSAGE_WITH_QUERY = "%s %s?%s";
-  private static final String MESSAGE_WITHOUT_QUERY = "%s %s";
   public static final org.sonar.api.utils.log.Logger Logger = Loggers.get("http");
 
-  private String contextRoot;
-  private Set<String> staticResourceDirs;
-
   @Override
   public void init(FilterConfig filterConfig) throws ServletException {
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/platform/web/RootFilter.java]: CRITICAL Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1186)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* CRITICAL 1 critical

Watch the comments in this conversation to review them.

**Comment by sns-seb** [comment]
> Documentation in conf/sonar.properties should be completed to explain to enable display of HTTP request ID in access.logs, else it's impossible to associate SQL and HTTP requests.

this is part of ticket SONAR-8332 not SONAR-8339. It's been done in another PR and merged (see https://github.com/SonarSource/sonarqube/commit/4a70f34b373f454367f1a2139b306470dcc012d2)

File: server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java [hunk 2]
```diff
@@ -73,23 +87,207 @@ public LoggerContextListener enableJulChangePropagation(LoggerContext loggerCont
...
+    return configureRootLogLevel(newLevel);
+  }
+
+  /**
+   * Configure the log level of the root logger to the specified level.
+   *
+   * @throws IllegalArgumentException if the specified level is not one of {@link #ALLOWED_ROOT_LOG_LEVELS}
+   */
+  public Level configureRootLogLevel(Level newLevel) {
+    Logger rootLogger = getRootContext().getLogger(ROOT_LOGGER_NAME);
```
Comment (SonarTech) [inline server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java]: MAJOR Make the "rootLogger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)

File: server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java [hunk 3]
```diff
@@ -73,23 +87,207 @@ public LoggerContextListener enableJulChangePropagation(LoggerContext loggerCont
...
+   *     the max number of files defined in property {@link #MAX_FILES_PROPERTY}</li>
+   *     <li>the logs will follow the specified log pattern</li>
+   *   </ul>
+   * </p>
+   *
+   * @see #buildLogPattern(RootLoggerConfig)
+   */
+  public FileAppender<ILoggingEvent> configureGlobalFileLog(Props props, RootLoggerConfig config, String logPattern) {
+    LoggerContext ctx = getRootContext();
+    Logger rootLogger = ctx.getLogger(ROOT_LOGGER_NAME);
```
Comment (SonarTech) [inline server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java]: MAJOR Make the "rootLogger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1312)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Pull request merged**